cmake/configure.cmake

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2016 Sebastian Weber, Henri Menke. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+configure_file(${IN} ${OUT})
```

Comment:
I'm relatively sure there is a better way...  I'll look into it.

Reply:
This actually seems to be the preferred solution. :confused: 
https://stackoverflow.com/questions/31057208/force-cmake-to-generate-configure-file-target-every-build
https://cmake.org/pipermail/cmake/2006-July/010105.html

Reply:
Yes, I also just came about this forum discussions.